starter-code/routes/celebrities.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express')
+
+const celebrityModel = require('../models/celebrity')
```

Comment:
same as in the other file. all models should start with uppercase (pascal casing). so: CelebrityModel